### PR TITLE
Add "My Recent Works" and "About Me" Sections

### PR DESCRIPTION
**Pull request by limavfabio**
This branch Adds "My Recent Works" and "About Me" Sections to the My-Portfolio project following HTML and CSS best practices and Microverse conventions.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -35,5 +35,113 @@ <h2 class="title">I'm a Software Developer</h2>
         <div class="medium"><svg width="24" height="24" xmlns="http://www.w3.org/2000/svg" fill-rule="evenodd" clip-rule="evenodd"><path d="M12 0c-6.626 0-12 5.372-12 12 0 6.627 5.374 12 12 12 6.627 0 12-5.373 12-12 0-6.628-5.373-12-12-12zm6.974 17.557v-.235l-1.092-1.072c-.096-.073-.144-.194-.124-.313v-7.874c-.02-.119.028-.24.124-.313l1.118-1.072v-.235h-3.869l-2.758 6.88-3.138-6.88h-4.059v.235l1.308 1.575c.128.115.194.285.176.457v6.188c.038.223-.032.451-.189.614l-1.471 1.784v.235h4.17v-.235l-1.471-1.784c-.158-.163-.233-.389-.202-.614v-5.352l3.66 7.985h.425l3.143-7.985v6.365c0 .17 0 .202-.111.313l-1.13 1.098v.235h5.49z"/></svg></div>
       </div>
   </section>
+  <section class="recent-works">
+    <div class="title">
+      <div class="text">My Recent Works</div>
```
Comment (omarsalem7) [inline index.html:40]: - [ ] Kindly use header tag `hx` to improve SEO.

File: index.html [hunk 2]
```diff
@@ -35,5 +35,113 @@ <h2 class="title">I'm a Software Developer</h2>
...
+      <div><img src="media/underline.svg" alt=""></div>
+    </div>
+    <div class="img-card-1">
+      <div class="main-image"><img src="media/Img-Placeholder.png" alt=""></div>
+      <div class="project-title">Multi-Post Stories</div>
```
Comment (omarsalem7) [inline index.html:45]: - [ ] As stated above use `hx` fro all title for the project to improve SEO.

File: index.html [hunk 3]
```diff
@@ -35,5 +35,113 @@ <h2 class="title">I'm a Software Developer</h2>
...
+      <div class="description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</div>
```
Comment (omarsalem7) [inline index.html:46]: - [ ] Kindly use `<p>` to build a good structure to improve SEO

File: index.html [hunk 4]
```diff
@@ -35,5 +35,113 @@ <h2 class="title">I'm a Software Developer</h2>
...
+      <div class="buttons">
+        <div class="simple-button">css</div>
+        <div class="simple-button">html</div>
+        <div class="simple-button">bootstrap</div>
+        <div class="simple-button">Ruby</div>
+        <div class="project-button">See Project</div>
+      </div>
```
Comment (omarsalem7) [inline index.html:53]: - [ ] This should be a list `ul` and each of them a `li` and for the rest projects

File: index.html [hunk 5]
```diff
@@ -35,5 +35,113 @@ <h2 class="title">I'm a Software Developer</h2>
...
+      <div class="buttons">
+        <div class="simple-button">html</div>
+        <div class="simple-button">bootstrap</div>
+        <div class="simple-button">Ruby</div>
+      </div>
+      <div class="project-button">See Project</div>
+    </div>
+  </section>
+  <section class="about-me">
+    <div class="title">About Me</div>
```
Comment (omarsalem7) [inline index.html:117]: - [ ] Kindly use `hx` as stated before for title

File: index.html [hunk 6]
```diff
@@ -35,5 +35,113 @@ <h2 class="title">I'm a Software Developer</h2>
...
+    <div class="description">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</div>
```
Comment (omarsalem7) [inline index.html:118]: - [ ] Kindly us `<p>` instead of `div` as stated before

File: index.html [hunk 7]
```diff
@@ -35,5 +35,113 @@ <h2 class="title">I'm a Software Developer</h2>
...
+    <div class="button">Get My Resume</div>
```
Comment (omarsalem7) [inline index.html:119]: - [ ] Kindly use button or `<a>` for buttons and follow the button styling: Active and Hover.
image

**Review by omarsalem7 — CHANGES_REQUESTED** [review]
Hi @limavfabio 😊, 

Good job so far! 👏
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there! 

you-are-near

## Highlights
- Descriptive pull request ✔️
- No linter errors ✔️
- Descriptive commit messages ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mmsesay — APPROVED** [review]
Hi @limavfabio, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**